tests/test_collectives.py

```diff
@@ -135,4 +140,4 @@ def test_collective(op: Callable[[torch.Tensor, int], torch.Tensor]) -> None:
 
 
 if __name__ == "__main__":
-    test_collective(pea.collectives.all_reduce_cross_replica_sum)
+    test_collective(pea.collectives.all_to_all_single_cross_replica)
```

Comment:
Seems a bit odd to have a `main` method in this test. Can this need be served by `./dev tests -k PATTERN`?

Reply:
Ah this was for me to debug tests. Can remove.